### ISSUE TITLE
CREATE COMPONENT:  Contact section

### ISSUE THREAD

**Issue by ProsperousHeart** [codefordallas/codefordallas.github.io#27]
- button with link to contact page (component)
- text:  `Have additional questions or need to contact us? Click the button to let us know on our contact page!`

**Comment by aditya123473892** [comment]
can i work on this issue

**Comment by ProsperousHeart** [comment]
Absolutely! Please only use regular HTML and CSS. Let us know when you're ready to integrate.

He really moving into a react app likely this week. After that, you won't have to check in with us on integration as we will have a contribution file that outlines next steps
